App.js

```diff
@@ -0,0 +1,29 @@
+const FigmaParser = require('./utils/FigmaParser');
```

Comment:
Remove `index.js` because is unused, then make the `App.js` the main file on package.json.

Also remove the `json-to-css.js` because it is just used on `index.js`

Reply:
Done removing the old files